### DISCUSSION TITLE
v1.0.1

### DISCUSSION THREAD

**Discussion by baffler** [baffler/Transparent-Twitch-Chat-Overlay#234] [General]
## Installation and Usage

- Download Installer: [TwitchChatOverlay-Setup.exe](https://github.com/baffler/Transparent-Twitch-Chat-Overlay/releases/download/v1.0.0/TwitchChatOverlay-Setup.exe)
- Once the app starts, you may be instructed to download the WebView2 Runtime
- You don't need to close or restart the app, just install the runtime from the link provided
- Use the menu button at the top-left to open settings
- Choose either KapChat, jChat, or Twitch Popout at the bottom of the settings window
- Enter the required information (typically just your username) and press Save.

## What's Changed

* Fixed issue with Zoom setting for custom/widget windows
* Customize Global Hotkeys in Settings -> General
* Changed default hotkeys to: Ctrl-Alt-F7 Toggles interaction, Ctrl-Alt-F8 Force app to be on top, Ctrl-Alt-F9 Toggle borders
* If you change the hotkeys, you should note that they are globally registered through Windows, so that key combination that you set for this app, can't be used within other applications while this application is running. So you'd typically want to pick a unique key combination with modifiers like Ctrl or Alt, or both.

<hr /><em>This discussion was created from the release <a href='https://github.com/baffler/Transparent-Twitch-Chat-Overlay/releases/tag/v1.0.1'>v1.0.1</a>.</em>

**Comment by Xs3roN** [comment]
Had to rever to 1.0 as 1.0.1 drops error as pasted below :/

---------------------------
Click on this message box and press Ctrl-C to copy the entire message
---------------------------
An unhandled exception occured. Note you can click on this message box to focus it, then press Ctrl-C to copy the entire message.

System.Windows.Markup.XamlParseException: Wywołanie konstruktora dla typu „TransparentTwitchChatWPF.MainWindow” zgodnego z określonymi ograniczeniami wiązania spowodowało wyjątek. ---> NHotkey.HotkeyAlreadyRegisteredException: Klawisz dostępu jest już zarejestrowany. (Wyjątek od HRESULT: 0x80070581) ---> System.Runtime.InteropServices.COMException: Klawisz dostępu jest już zarejestrowany. (Wyjątek od HRESULT: 0x80070581)

   --- Koniec śladu stosu wyjątków wewnętrznych ---

   w NHotkey.Hotkey.Register(IntPtr hwnd, String name) w C:\projects\nhotkey\src\NHotkey\Hotkey.cs:wiersz 52

   w NHotkey.HotkeyManagerBase.AddOrReplace(String name, UInt32 virtualKey, HotkeyFlags flags, EventHandler`1 handler) w C:\projects\nhotkey\src\NHotkey\HotkeyManagerBase.cs:wiersz 26

   w NHotkey.Wpf.HotkeyManager.AddOrReplace(String name, Key key, ModifierKeys modifiers, EventHandler`1 handler) w C:\projects\nhotkey\src\NHotkey.Wpf\HotkeyManager.cs:wiersz 115

   w TransparentTwitchChatWPF.MainWindow.SetupOrReplaceHotkeys()

   w TransparentTwitchChatWPF.MainWindow..ctor()

   --- Koniec śladu stosu wyjątków wewnętrznych ---

   w System.Windows.Markup.WpfXamlLoader.Load(XamlReader xamlReader, IXamlObjectWriterFactory writerFactory, Boolean skipJournaledProperties, Object rootObject, XamlObjectWriterSettings settings, Uri baseUri)

   w System.Windows.Markup.WpfXamlLoader.LoadBaml(XamlReader xamlReader, Boolean skipJournaledProperties, Object rootObject, XamlAccessLevel accessLevel, Uri baseUri)

   w System.Windows.Markup.XamlReader.LoadBaml(Stream stream, ParserContext parserContext, Object parent, Boolean closeStream)

   w System.Windows.Application.LoadBamlStreamWithSyncInfo(Stream stream, ParserContext pc)

   w System.Windows.Application.LoadComponent(Uri resourceLocator, Boolean bSkipJournaledProperties)

   w System.Windows.Application.DoStartup()

   w System.Windows.Application.<.ctor>b__1_0(Object unused)

   w System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)

   w System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)

   w System.Windows.Threading.DispatcherOperation.InvokeImpl()

   w System.Windows.Threading.DispatcherOperation.InvokeInSecurityContext(Object state)

   w MS.Internal.CulturePreservingExecutionContext.CallbackWrapper(Object obj)

   w System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)

   w System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)

   w System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)

   w MS.Internal.CulturePreservingExecutionContext.Run(CulturePreservingExecutionContext executionContext, ContextCallback callback, Object state)

   w System.Windows.Threading.DispatcherOperation.Invoke()

   w System.Windows.Threading.Dispatcher.ProcessQueue()

   w System.Windows.Threading.Dispatcher.WndProcHook(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)

   w MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)

   w MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)

   w System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)

   w System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)

   w System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)

   w MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)

   w MS.Win32.UnsafeNativeMethods.DispatchMessage(MSG& msg)

   w System.Windows.Threading.Dispatcher.PushFrameImpl(DispatcherFrame frame)

   w System.Windows.Threading.Dispatcher.PushFrame(DispatcherFrame frame)

   w System.Windows.Application.RunDispatcher(Object ignore)

   w System.Windows.Application.RunInternal(Window window)

   w System.Windows.Application.Run(Window window)

   w System.Windows.Application.Run()

   w TransparentTwitchChatWPF.App.Main()
---------------------------
OK   
---------------------------

**Reply by baffler** [reply]
If you're using multiple instances of the app, that's probably why. I fixed the issue in [v1.0.2](https://github.com/baffler/Transparent-Twitch-Chat-Overlay/releases/tag/v1.0.2) but I don't have a solution to have global hotkeys to work with the multiple instances yet. I can look into possibly adding it as a future fix.

**Reply by Xs3roN** [reply]
1.0.2 runs fine 👌

**Comment by brvnda** [comment]
ever since the update, i cant see any of my channel point redemptions. i've connected it to my twitch, enabled the community channel point redemptions, and it still doesn't work. is there any way to fix it? or is this just a bug? it use to work.
